oas/EmployeesApiBasic.yaml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.0
+info:
+  title: EmployeesApi
+  description: Employee management API for Db2.
+  version: "1.1"
```

Comment:
Worth setting this to 0.2 and the Starter version to 0.1 so we can show how the version number evolves, and keeping both < 1.1 which is the other example one that is more complete